spec/lib/insert/insert_card_spec.rb

```diff
@@ -55,6 +55,7 @@
 
   end
 
+
   describe 'for nonprofits' do
```

Comment:
This is testing inserting cards with nonprofits as a holder. We shouldn't need this anymore.